tool/tsh/tsh.go

```diff
@@ -3749,6 +3781,58 @@ func serializeAppsWithClusters(apps []appListing, format string) (string, error)
 	return string(out), trace.Wrap(err)
 }
 
+// TODO: Move this somewhere more appropriate
+const defaultSearchSessionPageLimit = 50
+
+func onSessions(cf *CLIConf) error {
+	fromUTC := time.Unix(0, 0)
+	toUTC := time.Now()
+	var err error
+	if cf.FromUTC != "" {
```

Comment:
A lot of this logic is the same between tctl and tsh, let's factor it out and reuse to avoid duplication.